src/main/java/com/hms_networks/americas/sc/thingworx/TWConnectorConsts.java

```diff
@@ -20,6 +20,13 @@
   public static final String CONNECTOR_VERSION =
       TWConnectorConsts.class.getPackage().getImplementationVersion();
 
+  /** Command to ensure the application always runs */
```

Comment:
There is library functionality to perform this task

https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/blob/main/src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppManagement.java

You will also want to add functionality to disable application auto restart in case
TWConnectorConsts.CONNECTOR_CONTROL_TAG_NAME is toggled to indicate the user is requesting the app to be shutdown which is just a call to SCAppManagement.disableAppAutoRestart()

Reply:
This avoids any issue with renaming the jar as well.

Reply:
Yeah... totally overlooked this.